### PR TITLE
SW-3072 small bugs in welcome wizard

### PR DESCRIPTION
**Pull request by khaledsherkawi**
SW-3072 Fix link

### CODE REVIEW

File: translations/messages.pot [hunk 1]
```diff
@@ -469,7 +469,7 @@ msgid ""
 msgstr ""
 
 #: octoprint_mrbeam/static/js/app/view-models/mrbeam.js:586
-msgid "Beta user: Please consider enabling Mr Beam Better Together!"
+msgid "Beta user: Please consider enabling Mr Beam analytics!"
```
Comment (irlaec) [inline translations/messages.pot:472]: Is this correct?

File: translations/messages.pot [hunk 2]
```diff
@@ -478,7 +478,7 @@ msgid ""
 "As you are currently in our Beta channel, you would help us tremendously "
 "sharing%(br)sthe laser job insights, so we can improve%(br)san overall "
 "experience working with the%(br)s Mr Beam. Thank you!%(br)s%(open)sGo to "
-"Better Together settings%(close)s"
+"analytics settings%(close)s"
```
Comment (irlaec) [inline translations/messages.pot:481]: and here

**Review by irlaec — CHANGES_REQUESTED** [review]

File: translations/messages.pot (hunk 1 above)
Comment (khaledsherkawi) [reply translations/messages.pot:472]: Yes, this was the discrepancy I talked about from the other branch so when the pot is re-generated, they showed up here. It is totally fine.

File: translations/messages.pot (hunk 2 above)
Comment (khaledsherkawi) [reply translations/messages.pot:481]: This as well

**Review by irlaec — DISMISSED** [review]

File: octoprint_mrbeam/translations/es/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -4926,11 +4933,15 @@ msgstr ""
...
+"name, login details, designs, or any other private content. You are in full "
+"control, and you can change your mind at any time by going to "
+"<strong>\"Settings > Better together\"</strong>."
 msgstr ""
 "Entendemos que tu privacidad es crucial, y queremos asegurarte que ninguno "
 "de los datos recogidos incluye tu información personal, como tu nombre, "
-"datos de acceso, diseños o cualquier otro contenido privado."
+"datos de acceso, diseños o cualquier otro contenido privado. Tú tienes todo "
+"el control, y puedes cambiar de opinión en cualquier momento yendo a "
+"<strong>“Ajustes > Mejor juntos”</strong>."
```
Comment (irlaec) [inline octoprint_mrbeam/translations/es/LC_MESSAGES/messages.po]: Oops, sorry... Just noticed that this wasn't changed after discussing with Cristina.
```suggestion
"<strong>“Ajustes > Mejorando juntos”</strong>."
```

**Review by irlaec — DISMISSED** [review]

**Review by irlaec — APPROVED** [review]

**Pull request merged**